nvml/Dockerfile

```diff
@@ -0,0 +1,39 @@
+# Use an intermediate step to build the nvml datadog integration
+# Setup taken from https://docs.datadoghq.com/agent/guide/community-integrations-installation-with-docker-agent/?tab=docker
+#
+# You will need to change this version to whichever datadog version you want to install
+# For example:
+#   docker build --build-arg=DD_AGENT_VERSION=7.18.0 .
+ARG DD_AGENT_VERSION=latest
+ARG PY_VERSION=3.8
+ARG INTEGRATIONS_VERSION=master
+
+FROM python:${PY_VERSION} AS wheel_builder
+ARG INTEGRATIONS_VERSION
+ARG DD_AGENT_VERSION
+WORKDIR /wheels
+
+RUN pip install "datadog-checks-dev[cli]==3.1.0"
+RUN git clone https://github.com/cresta/integrations-extras.git && cd integrations-extras && git status && git reset --hard ${INTEGRATIONS_VERSION}
```

Comment:
Happy to change this line on merge, but currently need it since this code isn't in `DataDog/integrations-extras` yet.